packages/webpack-build-scripts/webpack.config.base.js

```diff
@@ -38,27 +38,14 @@ const plugins = [
 ];
 if (IS_PRODUCTION) {
     plugins.push(
-        // Define JS globals when running the build
-        new webpack.DefinePlugin({
-            "process.env": {
-                NODE_ENV: JSON.stringify("production"),
-            },
-        }),
-
         // Some loaders accept configuration through webpack internals
         new webpack.LoaderOptionsPlugin({
             debug: false,
             minimize: true,
         }),
 
         // Only extract CSS to a file for production because it is slow
-        new ExtractTextPlugin("[name].css"),
-
-        // Minify JS
-        new UglifyJsPlugin(),
```

Comment:
when I did this migration on other projects, I switched to `terser-webpack-plugin`

Reply:
i am going to stick with webpack's built-in js minifier because these bundle files are never used by us.

Reply:
the built-in minifier doesn't work though, since they downgraded uglifyJS